feature/bitwarden/install.sh

```diff
@@ -0,0 +1,14 @@
+. /feature-installer-utils.sh
+
+if [ "X$(cat /home/cloudcontrol/flavour)X" == "XazureX" ] || [ "X$(cat /home/cloudcontrol/flavour)X" == "XsimpleX" ] || [ "X$(cat /home/cloudcontrol/flavour)X" == "XtanzuX" ]
+then
+  execHandle "Installing required libraries" sudo apk add gcompat
+fi
+
+TEMPDIR=$(mktemp -d)
+cd "${TEMPDIR}" || exit
+execHandle "Downloading bitwarden client" curl -L -o bw.zip "https://vault.bitwarden.com/download/?app=cli&platform=linux"
+execHandle "Unzipping bitwarden client" unzip bw.zip
+execHandle "Making bitwarden client executable" chmod +x bw
+execHandle "Installing bitwarden client" mv bw /home/cloudcontrol/bin
+rm -rf "${TEMPDIR}"
```

Comment:
Isn't the "cd - ..." missing at the end here?

Reply:
Fixed